### ISSUE TITLE
Error: is a non-GUI backend, so cannot show the figure.

### ISSUE THREAD

**Issue by ferdinandyb** [jktr/matplotlib-backend-kitty#14]
I'm using `python 3.11.1` with `ipython 8.8.0` and got this error on trying to plot:

```
[ins] In [6]: plt.show()
/home/fbence/Codes/matplotlib-backend-kitty/__init__.py:83: UserWarning: Matplotlib is currently using module://matplotlib-backend-kitty, which is a non-GUI backend, so cannot show the figure.
  _Backend.show(*args, **kwargs)
```

**Comment by ayorgo** [comment]
Please see #12.

**Comment by ferdinandyb** [comment]
Thanks! I didn't realize that is the same issue.

**Comment by jktr** [comment]
closed as duplicate of #12
